### PR TITLE
added doc/hardware

### PR DESCRIPTION
**Pull request by decarlof**
I will keep track here of the computing infrastructure used and ask the rest of the team to add the beamline instrumentation details.

### CODE REVIEW

**Comment by decarlof** [comment]
Please keep this open till I add more material in the hardware section

**Comment by decarlof** [comment]
Ok you can accept it for now. @vdeandrade can add beamline instrumentation details. @altaskier Chris do you have other computers at NW you want to list?

File: doc/source/hardware/doc.computing.rst [hunk 1]
```diff
@@ -1,4 +1,32 @@
...
+    
+name:  `xfm1 <http://www.supermicro.com/products/system/4U/4027/SYS-4027GR-TRT.cfm>`_
+
+    - CPU: 1 x Intel Xeon E5-2680 v2 Ten-Cores 2.80GHz
+    - RAM: 125GB Operating at 1866MHz Max (DDR3 DIMMs)
+    - GPU: 3 x Tesla K20c + 2 x Tesla K40
+    
+
++------------------+----------------+------------------+-------------+
+|    Features      |  Tesla K40     |    Tesls K20     |   units     |
```
Comment (dgursoy) [inline doc/source/hardware/doc.computing.rst:20]: Tests -> Tesla

**Comment by dgursoy** [comment]
I just saw this PR. Is it good to go?

**Comment by decarlof** [comment]
good to go

**Pull request merged**